src/validation.cpp

```diff
@@ -1825,6 +1825,8 @@ bool CChainState::ConnectBlock(const CBlock& block, CValidationState& state, CBl
     uint256 hashPrevBlock = pindex->pprev == nullptr ? uint256() : pindex->pprev->GetBlockHash();
     assert(hashPrevBlock == view.GetBestBlock());
 
+    nBlocksTotal++;
```

Comment:
uuuuhm, what is this change? I don't think it's supposed to touch validation at all!
I think you're doing an accidental reversion here.

Reply:
See #15283 for a description of the problem:

> During the loading of the genesis block, the bench print lines in ConnectTip divide by zero due to early return in ConnectBlock.

Do you mean it has been fixed elsewhere since being discussed in #15283?

Reply:
nBlocksTotal appears to be only referenced otherwise from bench output. And I assume what this is trying to do is avoid the genesis block early terminate from leaving the bench output with a zero.  I believe this is a reasonable change, though it will make bench results slightly incomparable (with small numbers of blocks processed) across versions.